application/models/Logbook_model.php

```diff
@@ -157,6 +163,49 @@ function create_qso() {
       $darc_dok = $this->input->post('darc_dok');
     }
 
+    $qso_locator = strtoupper(trim(xss_clean($this->input->post('locator')) ?? ''));
+    $qso_name = trim(xss_clean($this->input->post('name')));
+    $qso_age = null;
+    $qso_usa_state = $this->input->post('input_state_edit') == null ? '' : trim(xss_clean($this->input->post('input_state_edit'));
```

Comment:
@DJ3CE Missing a ) at the end of the line.

Reply:
:( Sorry for that.